dashboard/config/blocks/GamelabJr/gamelab_makeNewSprite.json

```diff
@@ -0,0 +1,27 @@
+{
+  "category": "Sprites",
+  "config": {
+    "color": [
+      184,
+      1,
+      0.74
+    ],
+    "func": "makeNewSprite",
+    "blockText": "make a new {ANIMATION} sprite at {X} {Y}",
```

Comment:
I think we can nuke this block in favor of Ryan's new create sprite block.  (Same for below.)

Reply:
These blocks are still used in the levels in /s/spritelab, I don't think we can nuke them just yet.